tests/test_cases/test_array/test_array.py

```diff
@@ -62,7 +62,7 @@ async def test_read_write(dut):
         _check_int(tlog, dut.param_char, ord('p'))
         _check_str(tlog, dut.param_str , b"ARRAYMOD")
 
-        if not cocotb.SIM_NAME.lower().startswith(("riviera")):
+        if not cocotb.SIM_NAME.lower().startswith("riviera"):
```

Comment:
Slightly unsure about that one -- we are trying to always pass a set here. But no strong feelings either way.

Reply:
`(value)` is equivalent to `value`. What should have been here was `(value,)` which is a tuple literal. But it is correct for now, `startswith` can take a single string or a tuple/list of strings.

Reply:
Indeed, I think this was a mistaken way of preparing for future extensions of that set.